actions/mysql_summary_action.go

```diff
@@ -0,0 +1,97 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package actions
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/percona/pmm/utils/pdeathsig"
+	"golang.org/x/sys/unix"
+)
+
+type mysqlSummaryAction struct {
```

Comment:
```suggestion
type ptMySQLSummaryAction struct {
```
please check everywhere

Reply:
rename the file

Reply:
@BupycHuk pt_mysql_summary_action.go and pt_mysql_summary_action_test.go?

Reply:
Renamed structure and files too.